### PR TITLE
Fix non-US spellings and the corresponding query

### PR DESCRIPTION
**Pull request by nickrolfe**
The query was missing multi-line comments containing 'analyse', because `.` does not match newline characters in `regexpMatch`.

### CODE REVIEW

**Review by jketema — COMMENTED** [review]
Small comment, otherwise C++ 👍

**Review by erik-krogh — APPROVED** [review]

**Review by jketema — DISMISSED** [review]

**Review by tausbn — APPROVED** [review]

File: ql/ql/src/codeql_ql/style/docs/NonUSSpellingQuery.qll
Comment (tausbn) [inline ql/ql/src/codeql_ql/style/docs/NonUSSpellingQuery.qll:21]: I wonder if this is faster or slower than using `[\\s\\S]` (IIRC)to match any character.

**Review by atorralba — APPROVED** [review]
Java 👍

**Comment by nickrolfe** [comment]
I just pushed a commit to fix the changenote formatting. Looks like I got approvals from all languages before I did that, so I'll merge as soon as I get one new approval (and CI checks pass).

**Review by erik-krogh — APPROVED** [review]

**Comment by nickrolfe** [comment]
Finally, the tests pass!

File: ql/ql/src/codeql_ql/style/docs/NonUSSpellingQuery.qll
Comment (nickrolfe) [reply ql/ql/src/codeql_ql/style/docs/NonUSSpellingQuery.qll:21]: I'm not sure, but I doubt it would be noticeable either way given the number of strings we're dealing with.

I'm about to gut this query to generalise it, and will likely replace this code, anyway.

**Comment by haaami01** [comment]
Thanks!  all 31 tests were successful✅👌

**Pull request merged**
